integrations/tensorflow/README.md

```diff
@@ -19,9 +18,43 @@ bazel build iree_tf_compiler:iree-import-xla
 bazel build iree_tf_compiler:iree-import-tf
 ```
 
-## Notes:
+Symlink binaries into python packages (only needs to be done once):
+
+```
+./symlink_binaries.sh
+```
+
+Pip install editable (recommend to do this in a virtual environment):
```

Comment:
What's "editable"? Also if they can pip install they shouldn't need to bazel build or symlink

Reply:
https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs

It's the usual way that Python packages are developed (if you aren't doing funny things, which we were previously). It symlinks them into your site-packages directory and changes in the source tree are immediately seen.

The only thing we need to build is the binaries that go into the packages, and we have to symlink them into the right place to make that work.